deploy/addons/coredns/coreDNS-clusterrole.yaml

```diff
@@ -16,3 +16,9 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
```

Comment:
Where did this part of the change come from? Is it a required part of the 1.2.4 update?

Reply:
@tstromberg This allows the federation plugin to fetch zone and region labels.
more details here: https://github.com/coredns/deployment/pull/100